src/wl/sys/wl_cfg80211_hybrid.c

```diff
@@ -106,14 +115,32 @@ static s32 wl_cfg80211_get_tx_power(struct wiphy *wiphy, struct wireless_dev *wd
 static s32 wl_cfg80211_get_tx_power(struct wiphy *wiphy, s32 *dbm);
 #endif
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 38)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 1, 0)
+static int wl_cfg80211_config_default_key(struct wiphy *wiphy,
+					   struct net_device *ndev, int link_id,
```

Comment:
The original variable name `dev` was renamed to `ndev` almost everywhere.
We should stick with the original name to keep the code maintainable and patches easy to cherry pick.